chain/chain/src/chain.rs

```diff
@@ -151,11 +151,15 @@ impl OrphanBlockPool {
     }
 
     fn add(&mut self, orphan: Orphan) {
-        let height_hashes =
-            self.height_idx.entry(orphan.block.header().height()).or_insert_with(|| vec![]);
+        let height_hashes = self
+            .height_idx
+            .entry(orphan.block.header().height())
+            .or_insert_with(std::vec::Vec::new);
```

Comment:
```suggestion
            .or_default();
```

It definitlly doesn't make sense to spell `std::vec::Vec::new` in full.